modules/Assay/Assay.yaml

```diff
@@ -338,6 +357,8 @@ enums:
         see_also:
           - Dynamic light scattering (DLS) is the ancestor of microrheology.
         source: https://www.elveflow.com/microfluidic-reviews/general-microfluidics/microrheology-a-review/
+      modified Skindex:
```

Comment:
Should we add Skindex too? In looking through I only found skindex mini : https://bioportal.bioontology.org/ontologies/LOINC?p=classes&conceptid=http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FLNC%2FLP420590-4

Reply:
Skindex was found to be insufficient for use in NF patients. I am guessing we do not need to add it because people will probably not use the normal skindex. 

There are several "modified" flavors of it (e.g. cNF-skindex). I'm not sure if any of the modified flavors are available in any ontologies.

Reply:
I will check with Sang to see if these investigators are using cNF skindex or something else.

Reply:
Turns out they are using cNF-skindex and skindex-16 for this study, so I got rid of the "modified" and added these two.